apps/web/src/AppRoutes.tsx

```diff
@@ -124,6 +125,7 @@ export const AppRoutes = () => {
           <Route path="" element={<LayoutsListPage />} />
         </Route>
         <Route path="/translations/*" element={<TranslationRoutes />} />
+        <Route path="/docs/*" element={<DocsPage />} />
```

Comment:
@davidsoderberg how is this going to work? Why do we need a new page for docs if it's accessed via a side menu?

Reply:
Right now I am only doing this to be able to implement the docs component until I know how we will display it inside of the web. Radek and Nik will sync about this :)